packages/cli/src/commands/init/init.js

```diff
@@ -90,6 +93,90 @@ async function createFromExternalTemplate(
   }
 }
 
+async function installPods(loader: typeof Ora) {
+  process.chdir('ios');
+
+  const hasPods = await fs.pathExists('Podfile');
+
+  if (!hasPods) {
+    return;
+  }
+
+  try {
+    await commandExists('pod');
+  } catch (err) {
+    loader.succeed();
+
+    const {shouldInstallCocoaPods} = await inquirer.prompt([
+      {
+        type: 'confirm',
+        name: 'shouldInstallCocoaPods',
+        message: 'CocoaPods is not installed, Do you want to install it?',
+      },
+    ]);
+
+    if (shouldInstallCocoaPods) {
+      try {
+        await execa('sudo', ['gem', 'install', 'cocoapods'], {
+          stdio: 'pipe',
+        });
+      } catch (err) {
+        throw new CLIError(
+          'Error occurred while trying to install CocoaPods, please install it manually.',
+          err,
+        );
+      }
+
+      try {
+        loader.start('Installing pods');
+      } catch (err) {
+        throw new CLIError(
+          'Failed to run "pod install", please try to run it manually.',
+          err,
+        );
+      }
+    }
+  }
+
+  try {
+    await execa('pod', ['install'], {
+      stdio: 'pipe',
+    });
+  } catch (err) {
+    throw new CLIError(
+      'Failed to run "pod install", please try to run it manually.',
+      err,
+    );
+  }
+}
+
+async function installDependencies({
+  npm,
+  loader,
+}: {
+  npm?: boolean,
+  loader: typeof Ora,
+}) {
+  loader.start('Installing all required dependencies');
+
+  await PackageManager.installAll({
+    preferYarn: !npm,
+    silent: true,
+  });
+
+  try {
+    await installPods(loader);
+
```

Comment:
nit: since we change `chdir` inside `installPods`, I would also bring it back inside it and here, just call `await installPods` w/o any try/catch.

Feels cleaner

Reply:
Hmm, if we do this:

```
await installPods(loader);

process.chdir('..');

loader.succeed();
```

The `process.chdir('..');` will never be called if `installPods` fails, is that what you mean?

Reply:
Good one, fixed.